src/borg/archiver.py

```diff
@@ -4104,15 +4104,16 @@ def define_borg_mount(parser):
                                         '"borg help compression" command for details.')
         archive_group.add_argument('--recompress', metavar='MODE', dest='recompress', nargs='?',
                                    default='never', const='if-different', choices=('never', 'if-different', 'always'),
-                                   help='recompress data chunks according to ``--compression``. '
-                                        'MODE `if-different`: '
-                                        'recompress if current compression is with a different compression algorithm '
-                                        '(the level is not considered). '
-                                        'MODE `always`: '
-                                        'recompress even if current compression is with the same compression algorithm '
-                                        '(use this to change the compression level). '
-                                        'MODE `never` (default): '
-                                        'do not recompress.')
+                                   help='recompress data chunks according to `MODE` and ``--compression``. '
+                                        'Possible modes are '
+                                        '`if-different`: recompress if current compression is with a different '
+                                        'compression algorithm (the level is not considered); '
+                                        '`always`: recompress even if current compression is with the same '
+                                        'compression algorithm (use this to change the compression level); and '
+                                        '`never`: do not recompress (use this option to explicitly prevent '
+                                        'recompression). '
+                                        'If no MODE is given, `if-different` will be used. '
+                                        'Not passing --recompress is equivalent "--recompress never".')
```

Comment:
... is equivalent **to** ...

Yeah, much better now. That should not leave any unclarities.

Reply:
Well spotted. Fixed